### ISSUE TITLE
depending on mvn command illegal transitive dependencies are found and not found 

### ISSUE THREAD

**Issue by mbiarnes** [ImmobilienScout24/illegal-transitive-dependency-check#25]
If you mvn clean install you will see that no illegal transitive dependencies are found
but if you do mvn gwt:run-codeserver you will see a lot of illegal transitive dependencies.

He re is a link where you can download a web-app: 
https://github.com/droolsjbpm/guvnor/tree/master/guvnor-webapp

**Comment by 2bad4u** [comment]
After some struggle with GWT and a downgrade to JDK 7 I was able to build your project. As far as I understand the GWT build fetches a lot of dependencies at compile time (which are not more necessary during runtime). After adding these dependencies the check does not further complain:
```
org.drools:drools-workbench-models-datamodel-api
org.uberfire:uberfire-commons-editor-api
org.jboss.errai:errai-marshalling
org.dashbuilder:dashbuilder-dataset-api
org.uberfire:uberfire-runtime-plugins-api
org.kie.uberfire:kie-uberfire-social-activities-api
org.uberfire:uberfire-widgets-properties-editor-api
org.uberfire:uberfire-widgets-service-api
org.dashbuilder:dashbuilder-displayer-api
org.uberfire:uberfire-backend-api
org.uberfire:uberfire-apps-api
```
So far I see only to possible ways to cope with that problem: either disabling the check for that particular module or declaring all the dependencies above (which may be a bit verbose in your case)

**Comment by mbiarnes** [comment]
thanX André,
We did the 2nd way.

Michael

----- Ursprüngliche Mail -----
Von: "André Schubert" <notifications@github.com>
An: "ImmobilienScout24/illegal-transitive-dependency-check" <illegal-transitive-dependency-check@noreply.github.com>
CC: "mbiarnes" <mbiarnes@redhat.com>
Gesendet: Donnerstag, 29. Januar 2015 19:20:01
Betreff: Re: [illegal-transitive-dependency-check] depending on mvn command illegal transitive dependencies are found and not found  (#25)

After some struggle with GWT and a downgrade to JDK 7 I was able to build your project. As far as I understand the GWT build fetches a lot of dependencies at compile time (which are not more necessary during runtime). After adding these dependencies the check does not further complain:
```
org.drools:drools-workbench-models-datamodel-api
org.uberfire:uberfire-commons-editor-api
org.jboss.errai:errai-marshalling
org.dashbuilder:dashbuilder-dataset-api
org.uberfire:uberfire-runtime-plugins-api
org.kie.uberfire:kie-uberfire-social-activities-api
org.uberfire:uberfire-widgets-properties-editor-api
org.uberfire:uberfire-widgets-service-api
org.dashbuilder:dashbuilder-displayer-api
org.uberfire:uberfire-backend-api
org.uberfire:uberfire-apps-api
```
So far I see only to possible ways to cope with that problem: either disabling the check for that particular module or declaring all the dependencies above (which may be a bit verbose in your case)


---
